powerhome/playbook
# Dialog title patch

#### Screens

It was not possible to use the dialog kit without sending a `title` prop:

![image](https://user-images.githubusercontent.com/8194056/185212504-67ac0045-024d-404b-8f88-80f72b4763c5.png)

Now you can use it only with `text`:
<img width="1236" alt="Screen Shot 2022-08-17 at 3 12 48 PM" src="https://user-images.githubusercontent.com/8194056/185212406-5fc3e5e3-3abb-473f-9151-527c65bd3a97.png">

#### Breaking Changes

No breaking changes, fix bugs only. 

#### Runway Ticket URL

[INSERT URL]

#### How to test this

Create a Kit without sending the `title` prop. Make sure you double check all the dialogs in order to assure quality.

#### Checklist:
